ProjectManagmentSystem/src/Modules/DashboardModule/components/Dashboard/Dashboard.tsx

```diff
@@ -1,7 +1,159 @@
-import React from 'react'
+import React, { useContext, useEffect, useState } from "react";
+import Styles from "./Dashboard.module.css";
+import axios from "axios";
+import { AuthContext } from "../../../Context/AuthContext";
 
 export default function Dashboard() {
+  const [usersList, setUsersList] = useState([]);
+  const [projectsList, setProjectsList] = useState([]);
+  const [tasksList, setTasksList] = useState([]);
+  const { requestHeaders, baseUrl }: any = useContext(AuthContext);
+
+  const getUsersList = async () => {
+    try {
+      let response = await axios.get(`${baseUrl}/Users/manager`, {
+        headers: requestHeaders,
+      });
+      setUsersList(response.data.data);
+
+      console.log("Number try", response.data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const getProjectsList = async () => {
+    try {
+      let response = await axios.get(`${baseUrl}/Project/manager`, {
+        headers: requestHeaders,
+      });
+      setProjectsList(response.data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const getTasksList = async () => {
+    try {
+      let response = await axios.get(`${baseUrl}/Task/manager`, {
+        headers: requestHeaders,
+      });
+      setTasksList(response.data.data);
+    } catch (error: any) {}
+  };
+  useEffect(() => {
+    getUsersList();
+    getProjectsList();
+    getTasksList();
+  }, []);
+  const countUsersByActivation = (users: any) => {
```

Comment:
There are two APIs for the dashboard:
Task/Count
Users/Count

It will provide all needed information for the charts.

Reply:
![image](https://github.com/KareemTarekHussin/PMS/assets/64327685/1e6a6e8c-b5ca-4179-ba5d-c2d726b1cf63)

Reply:
![image](https://github.com/KareemTarekHussin/PMS/assets/64327685/bf005e21-85a8-4b0a-bcfb-9c2c588cf0dd)

Reply:
This was Not for the charts it was for the two divs under the header as we needed to change user status